### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sabryalani**
Add K-Net Logo

### CODE REVIEW

**Comment by andrewpaliga** [comment]
The logo size will need to be changed to match specifications: 

>Add your-icon.svg to the app/assets/images/payment_icons/ directory with the following specifications:
38 by 24 pixels
Solid white background

**Comment by sabryalani** [comment]
Hello Andrew,

 

Please be informed that requested logo was uploaded and the PR was modified accordingly.

 

Waiting for your response.

 

 


Thanks & Regards

 

 

 	

 

 

 	



Sabry Alani

 	

Senior Software Enginner

 	

E: s.alani@tap.com.kw

 	

M: +965 60016692

 	

T: +965 22922854

 	

F: +965 22922880

 	

 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 	

 

 

 	

Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

**Comment by sabryalani** [comment]
Hello Andrew,

 

Any update regarding our request?

 

 


Thanks & Regards

 

 

 	

 

 

 	



Sabry Alani

 	

Senior Software Enginner

 	

E: s.alani@tap.com.kw

 	

M: +965 60016692

 	

T: +965 22922854

 	

F: +965 22922880

 	

 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 	

 

 

 	

Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

 

 

From: Sabry Alani [mailto:s.alani@tap.com.kw] 
Sent: Sunday, August 21, 2016 10:22 AM
To: 'activemerchant/payment_icons' <reply@reply.github.com>; 'activemerchant/payment_icons' <payment_icons@noreply.github.com>
Cc: 'Author' <author@noreply.github.com>
Subject: RE: [activemerchant/payment_icons] Add files via upload (#18)

 

Hello Andrew,

 

Please be informed that requested logo was uploaded and the PR was modified accordingly.

 

Waiting for your response.

 

 


Thanks & Regards

 

 

 


 

 

 




Sabry Alani

 


Senior Software Enginner

 


E: s.alani@tap.com.kw <mailto:s.alani@tap.com.kw> 

 


M: +965 60016692

 


T: +965 22922854

 


F: +965 22922880

 


 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 


 

 

 


Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

**Comment by sabryalani** [comment]
Hello Andrew,

 

Any update regarding our request?

 

 


Thanks & Regards

 

 

 


 

 

 




Sabry Alani

 


Senior Software Enginner

 


E: s.alani@tap.com.kw <mailto:s.alani@tap.com.kw> 

 


M: +965 60016692

 


T: +965 22922854

 


F: +965 22922880

 


 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 


 

 

 


Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

 

 

From: Sabry Alani [mailto:s.alani@tap.com.kw] 
Sent: Sunday, August 21, 2016 10:22 AM
To: 'activemerchant/payment_icons' <reply@reply.github.com>; 'activemerchant/payment_icons' <payment_icons@noreply.github.com <mailto:payment_icons@noreply.github.com> >
Cc: 'Author' <author@noreply.github.com <mailto:author@noreply.github.com> >
Subject: RE: [activemerchant/payment_icons] Add files via upload (#18)

 

Hello Andrew,

 

Please be informed that requested logo was uploaded and the PR was modified accordingly.

 

Waiting for your response.

 

 


Thanks & Regards

 

 

 


 

 

 




Sabry Alani

 


Senior Software Enginner

 


E: s.alani@tap.com.kw <mailto:s.alani@tap.com.kw> 

 


M: +965 60016692

 


T: +965 22922854

 


F: +965 22922880

 


 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 


 

 

 


Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

**Comment by sabryalani** [comment]
Hello dears,

 

Any update regarding the below request request?

 

 


Thanks & Regards

 

 

 


 

 

 




Sabry Alani

 


Senior Software Enginner

 


E: s.alani@tap.com.kw <mailto:s.alani@tap.com.kw> 

 


M: +965 60016692

 


T: +965 22922854

 


F: +965 22922880

 


 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 


 

 

 


Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

 

 

From: Sabry Alani [mailto:s.alani@tap.com.kw] 
Sent: Sunday, August 21, 2016 10:22 AM
To: 'activemerchant/payment_icons' <reply@reply.github.com>; 'activemerchant/payment_icons' <payment_icons@noreply.github.com <mailto:payment_icons@noreply.github.com> >
Cc: 'Author' <author@noreply.github.com <mailto:author@noreply.github.com> >
Subject: RE: [activemerchant/payment_icons] Add files via upload (#18)

 

Hello Andrew,

 

Please be informed that requested logo was uploaded and the PR was modified accordingly.

 

Waiting for your response.

 

 


Thanks & Regards

 

 

 


 

 

 




Sabry Alani

 


Senior Software Enginner

 


E: s.alani@tap.com.kw <mailto:s.alani@tap.com.kw> 

 


M: +965 60016692

 


T: +965 22922854

 


F: +965 22922880

 


 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 


 

 

 


Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

**Comment by andrewpaliga** [comment]
@sabryalani your logos still do not match the size requirements. The pull request sill need to look very similar to [this one](https://github.com/activemerchant/payment_icons/commit/23fd7b244cf8d7252cb78e0f51f69eb137b3cd23).

It looks like you have multiple logos on the TAP.svg. This is not permitted.

**Comment by sabryalani** [comment]
Hello Andrew,

I have modified the files as requested, please find below the PR URL:
https://github.com/sabryalani/payment_icons/pull/1/files

 

Waiting for your feedback.

 

 


Thanks & Regards

 

 

 	

 

 

 	



Sabry Alani

 	

Senior Software Enginner

 	

E: s.alani@tap.com.kw

 	

M: +965 60016692

 	

T: +965 22922854

 	

F: +965 22922880

 	

 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 	

 

 

 	

Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

File: db/payment_icons.yml [hunk 1]
```diff
@@ -103,3 +103,7 @@
   label: NetBanking
   group: bank_transfers
 
+- 
+  name: K-Net
```
Comment (AnotherJoSmith) [inline db/payment_icons.yml]: name needs to be all lowercase. This should be `name: k-net`

File: app/assets/images/payment_icons/k-net-logo.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AnotherJoSmith) [inline app/assets/images/payment_icons/k-net-logo.svg]: The file name needs to be `k-net.svg`

**Comment by AnotherJoSmith** [comment]
As @andrewpaliga mentioned, the logo needs to be 38 by 24 pixels. Your logo is currently 243x152 pixels.

**Comment by sabryalani** [comment]
Dear Jonathan,

 

We’ve modified the files as requested.

 

PR URL: https://github.com/sabryalani/payment_icons/pull/1/files

 

 


Thanks & Regards

 

 

 	

 

 

 	



Sabry Alani

 	

Senior Software Enginner

 	

E: s.alani@tap.com.kw

 	

M: +965 60016692

 	

T: +965 22922854

 	

F: +965 22922880

 	

 <http://www.gotapnow.com/> www.gotapnow.com
P.O Box 3908, Safat 13040, Kuwait

 	

 

 

 	

Notice & Disclaimer
This e-mail and any files transmitted with it are intended solely for the use of the individual or entity to which they are addressed and may be legally privileged and/or confidential. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any dissemination or copying of this e-mail or any of its attachment(s) is strictly prohibited. Any unauthorized use or disclosure may be unlawful. If you have received this e-mail in error, please immediately notify the sending individual or entity by e-mail and permanently delete the original e-mail and attachment(s) from your computer system. Any opinions contained in this message are those of the author and are not given or endorsed by TAP unless otherwise clearly indicated in the message, and the authority of the author to so bind TAP is duly verified. No warranty is given as to the legitimacy of email content transmitted over the Internet. Thank you for your cooperation.

File: db/payment_icons.yml [hunk 3]
```diff
@@ -103,3 +103,7 @@
   label: NetBanking
   group: bank_transfers
```
Comment (andrewpaliga) [inline db/payment_icons.yml]: Can you remove this white space?

**Comment by sabryalani** [comment]
Hello dear,


Thanks a lot for your email.


Please be informed that I’m currently on vacation with limited internet access till Sep. 25 2016.


For urgent assistance kindly contact Mr. Kalai Murugan on:  t.kalai@tap.com.kw  


Appreciate your cooperation and understanding

**Comment by andrewpaliga** [comment]
Looks good. If you can just remove the white space I commented on I'll be able to merge.

**Comment by millingab** [comment]
@sabryalani can you merge the whitespace ?

**Comment by sabryalani** [comment]
hi @andrewpaliga, white spaces removed. 

can you please advise if all ok now.

**Comment by NathanPJF** [comment]
Going to close this one from inactivity.

**Pull request closed without merging**